### PR TITLE
[ASSIST-455] Task Management Updates

### PR DESCRIPTION
**Pull request by mxm1070**
https://engage.canyongbs.com/workgroups/group/3/tasks/task/view/455/

### CODE REVIEW

**Review by Orrison — CHANGES_REQUESTED** [review]
This looks great! If you could please update the Kanban view to display the task description in the card as well? Replacing the description dummy text I had there.

You will probably want to truncate that after a reasonable length.

File: app-modules/task/src/Filament/Concerns/TaskViewActionInfoList.php [hunk 1]
```diff
@@ -19,47 +19,43 @@ trait TaskViewActionInfoList
...
+                            ->columnSpanFull(),
+                        TextEntry::make('description')
+                            ->columnSpanFull(),
+                        TextEntry::make('assignedTo.name')
+                            ->label('Assigned To')
+                            ->url(fn (Task $record) => $record->assignedTo ? UserResource::getUrl('view', ['record' => $record->assignedTo]) : null)
+                            ->default('Unassigned'),
+                        TextEntry::make('concern.display_name')
+                            ->label('Concern')
+                            ->getStateUsing(fn (Task $record) => $record->concern->{$record->concern::displayNameKey()})
```
Comment (dgoetzit) [inline app-modules/task/src/Filament/Concerns/TaskViewActionInfoList.php:35]: We'll need to account for a potentially `null` concern here.

**Review by dgoetzit — CHANGES_REQUESTED** [review]
Looks good! Just one minor change on my end.

**Comment by mxm1070** [comment]
> This looks great! If you could please update the Kanban view to display the task description in the card as well? Replacing the description dummy text I had there.
> 
> You will probably want to truncate that after a reasonable length.

Done, and I set the limit to 50 characters.

**Review by Orrison — APPROVED** [review]

**Pull request merged**